packages/playwright-core/src/server/registry/index.ts

```diff
@@ -36,6 +36,12 @@ export { writeDockerVersion } from './dependencies';
 const PACKAGE_PATH = path.join(__dirname, '..', '..', '..');
 const BIN_PATH = path.join(__dirname, '..', '..', '..', 'bin');
 
+const PLAYWRIGHT_CDN_MIRRORS = [
```

Comment:
Let's consider adding install tests that induce testing fallback CDNs! Sort of like testing to make sure a database backup can actually be used if needed! :) 

Maybe a separate PR?

Reply:
I'll follow up with the test! https://github.com/microsoft/playwright/issues/14958

❤️ 😄 🎭